kubespawner/utils.py

```diff
@@ -27,3 +27,112 @@ def generate_hashed_slug(slug, limit=63, hash_length=6):
         prefix=slug[:limit - hash_length - 1],
         hash=slug_hash[:hash_length],
     ).lower()
+
+
+def update_k8s_model(target, source, logger=None, origin=None):
```

Comment:
rename `origin` to `target_name`, it would make it easier to understand the parameters function